libs/community/langchain_community/chat_models/yandex.py

```diff
@@ -32,16 +32,15 @@ def _parse_chat_history(history: List[BaseMessage]) -> Tuple[List[Dict[str, str]
         A tuple of a list of parsed messages and an instruction message for the model.
     """
     chat_history = []
-    instruction = ""
     for message in history:
         content = cast(str, message.content)
         if isinstance(message, HumanMessage):
             chat_history.append(_parse_message("user", content))
         if isinstance(message, AIMessage):
             chat_history.append(_parse_message("assistant", content))
         if isinstance(message, SystemMessage):
-            instruction = content
-    return chat_history, instruction
+            chat_history.append(_parse_message("system", content))
```

Comment:
Hi @tyumentsev4 Is it possible to add ut-s? The changes are quite heavy.

Reply:
Hi @leo-gan, this pull request proposes to stop using the deprecated API v1alpha. Go to use  [Foundation Models API](https://cloud.yandex.com/en/docs/yandexgpt/api-ref/v1/grpc/).

Reply:
Got it. Then it is ok.